src/renderer/login/components/Login/Login.js

```diff
@@ -18,7 +18,7 @@ const TAB_FILE = 'file';
 
 const TABS = {
   [TAB_PASSPHRASE]: 'Passphrase',
-  [TAB_WIF]: 'WIF',
+  [TAB_WIF]: 'Private Key',
```

Comment:
I think for consistency, we should rename `TAB_WIF` to `TAB_PRIVATE_KEY`, as well as renaming `LoginFormWIF` to `LoginFormPrivateKey`.